src/OpenTelemetry.Exporter.Prometheus.HttpListener/PrometheusHttpListener.cs

```diff
@@ -22,6 +22,8 @@ namespace OpenTelemetry.Exporter;
 
 internal sealed class PrometheusHttpListener : IDisposable
 {
+    private const string OpenMetricsMediaType = "application/openmetrics-text";
```

Comment:
This could be moved to the common `PrometheusExporter` class instead of both `PrometheusHttpListener` and `PrometheusExporterMiddleware` define it.

Reply:
Created `PrometheusHeadersParser` that encapsulates all the header logic and shared it with both projects.